_ml-commons-plugin/api/model-apis/undeploy-model.md

```diff
@@ -56,3 +56,35 @@ POST /_plugins/_ml/models/_undeploy
   }
 }
 ```
+### Automatically undeploy a model based on TTL
+
+Starting with OpenSearch version 2.14, models can be automatically undeployed from memory based on the predefined time-to-live (TTL) when the model was last accessed or used. To define a TTL that auto undeploys a model, include the following ModelDeploySetting in your ML mode. Please note that model TTLs are checked periodically by a syn_up cron job, so the maximum time a model lives in the memory could be TTL + sync_up_job_interval. The default cron job interval is 10 seconds. To update the cron job internal, use the following cluster setting.
```

Comment:
Check the `GitHub Actions / style-job` 
> Using 'Please' is unnecessary. Remove

Reply:
If I'm understanding correctly, the user can indeed update the model's Time To Live (TTL) after it has been deployed without encountering an exception. However, this TTL update will not take effect immediately. A deployment or redeployment of the model is required for the updated TTL to come into effect.